open-craft/xblock-chat
# BB-1358 Add i18n tools config, transifex and automatization

This PR adds the necessary configuration for edx's i18n tools and transifex. it also adds the necessary maketargets to maintain translations. 

**JIRA tickets**: 

**Discussions**:

**Dependencies**: None

**Screenshots**: 
<img width="1011" alt="Screen Shot 2019-06-14 at 11 27 17 AM" src="https://user-images.githubusercontent.com/4007280/59523639-6c518b00-8e97-11e9-9e92-d1cd16e64a2f.png">


**Sandbox URL**: TBD - sandbox is being provisioned.

**Merge deadline**: "None" if there's no rush, "ASAP" if it's critical, or provide a specific date if there is one.

**Testing instructions**:

1. Checkout this branch
2. Change edx language to spanish
3. Check the messages are not in english.

**Author notes and concerns**:

- Not really sure if the transifex configuration is correct since I'm not sure how to add a repo there.
